### ISSUE TITLE
Removing old files from OP3 experiment

### ISSUE THREAD

**Issue by jpcima** [Wohlstand/libADLMIDI#185]
Inside the banks folder is a leftover binary file `op3_files/2x2patch/bytecmp` (from 67824478db7965bb98072ca28d9ffaf347b4f6ab).
This triggers the debian linter in this way:

```
P: adlplug source: source-contains-prebuilt-binary thirdparty/libADLMIDI/fm_banks/op3_files/2x2patch/bytecmp
N: 
N:    The source tarball contains a prebuilt ELF object. They are usually left
N:    by mistake when generating the tarball by not cleaning the source
N:    directory first. You may want to report this as an upstream bug, in case
N:    there is no sign that this was intended.
N:    
N:    Severity: pedantic, Certainty: certain
N:    
N:    Check: cruft, Type: source
```

I think it's safe to remove this file, what about others? I'm pretty sure `res.txt` also can go.
What of `bytecmp.c`, has it any purpose at this point?

**Comment by Wohlstand** [comment]
Those files I made while reversed bank format from this driver
https://bitbucket.org/djtubig-malicex/opl3-synth-driver/src/master/
to be able to import bank into libADLMIDI (at that moment, I had OP3 only as best choice until I have added support for AIL and later when I have invented WOPL).

So, the purpose of those files is only historical memory. They are freely can be removed away with no any destruction of working stuff. The bank I have took is now №68 and was modded & improved by me which now gives better sound than originally.
